ludwig/benchmarking/summary_dataclasses.py

```diff
@@ -31,7 +35,7 @@ class ExperimentSummary:
 
 
 @dataclass
-class MetricDiff:
+class Diff:
```

Comment:
`MetricDiff` is more descriptive than `Diff`

Reply:
I would rename this to `MetricDiff`.

`Diff` is rather generic, and the global namespace doesn't make this clearer, as there's other "Diff" dataclasses in the same file.

```python
from ludwig.benchmarking.summary_dataclasses import Diff  # seems ambiguous.
```